edge_orchestrator/src/edge_orchestrator/interface/api/routers/v1/router.py

```diff
@@ -127,6 +129,30 @@ async def trigger_job(
     return {"item_id": item.id}
 
 
+async def data_gathering_job(
+    class_name: str = None,
+    binaries: List[UploadFile] = [],
+    cameras_metadata: Dict[str, CameraConfig] = {},
+    data_gathering: DataGathering = Depends(get_data_gathering),
+    station_config: StationConfig = Depends(get_config),
+    background_tasks: BackgroundTasks = None,
+):
+    print("class_name", class_name)
+    # Set the class name on the config
+    station_config.binary_storage_config.class_directory = class_name
+    station_config.metadata_storage_config.class_directory = class_name
+
+    input_binaries = {}
+    for binary in binaries:
+        input_binaries[binary.filename] = Image(image_bytes=binary.read())
```

Comment:
je pense qu'il y a besoin de mettre un await car UploadFile est async : 
```suggestion
        input_binaries[binary.filename] = Image(image_bytes=await binary.read())
```

Reply:
si tu peux le rajouter aussi côté route trigger du supervisor stp